GVRf/Framework/src/org/gearvrf/GVRSceneObject.java

```diff
@@ -143,29 +143,27 @@ public GVRSceneObject(GVRContext gvrContext, GVRMesh mesh,
      * that, say, {@code sceneObject2} and {@code sceneObject3} are using the
      * same mesh as {@code sceneObject1}, and will only load the mesh once.
      * 
-     * @param context
+     * @param gvrContext
      *            current {@link GVRContext}.
-     * @param mesh
-     *            Basically, a stream containing a mesh file.
-     * @param texture
-     *            Basically, a stream containing a texture file. This can be
-     *            either a compressed texture or a regular Android bitmap file.
-     *            
-     * @since 1.6.7            
+     * @param futureMesh
+     *            mesh of the object.
+     * @param futureTexture
+     *            texture of the object.
+     * 
+     * @since 1.6.7
```

Comment:
But this is not true ... I think you should probably add a `GVRVersion.V_1_6_8` ...